File: 프로그래머스/lv2/12924. 숫자의 표현/숫자의 표현.py
def solution(n):
    
    cnt=0
    for i in range(1,n//2+1):
        sum=0
        for j in range(i,n):
            sum+=j
            if sum==n:
                cnt+=1
                break;
            if sum>n:
                break;

    return cnt+1

File: 프로그래머스/lv2/12924. 숫자의 표현/숫자의 표현.java
class Solution {
    public int solution(int n) {
        int cnt=0;
        for(int i=1;i<=n/2;i++){
            int sum=0;
            for(int j=i;j<=n/2+1;j++){
                sum+=j;
                if(sum>n) break;
                if(sum==n){
                    cnt++;
                    break;
                }
            }
        }
        return cnt+1;
    }
}